File: Submissions/#0412_div2/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);

    int n = sc.nextInt();

    int[] rate = new int[n];

    String answer = "maybe";

    for (int i = 0; i < n; i++) {

      int a = sc.nextInt();
      int b = sc.nextInt();
  
      if (a != b) {
        answer = "rated";
      } else {
        rate[i] = a;
      }
  
    }

    sc.close();
  
    if (answer == "maybe") {
      for (int i = 0; i < n; i++) {
        for (int j = i + 1; j < n; j++) {
          if (rate[i] < rate[j]) {
            answer = "unrated";
            break;
          }
        }
      }
    }
    
    System.out.println(answer);

  }

}

File: Submissions/#0412_div2/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int n;
  cin >> n;

  vector<int> rate(n);

  string answer = "maybe";

  for (int i = 0; i < n; i++) {

    int a, b;
    cin >> a >> b;

    if (a != b) {
      answer = "rated";
    } else {
      rate[i] = a;
    }

  }

  if (answer == "maybe") {
    for (int i = 0; i < n; i++) {
      for (int j = i + 1; j < n; j++) {
        if (rate[i] < rate[j]) {
          answer = "unrated";
          break;
        }
      }
    }
  }

  cout << answer << endl;

}